Quadriga/src/main/java/edu/asu/spring/quadriga/service/transformation/impl/TransformationManager.java

```diff
@@ -51,4 +55,80 @@ public void saveTransformations(ITransformationFile transformationFile) throws F
         return transformationDAO.getAllTransformations();
     }
 
+    /**
+     * Retrieve the transformation file object which contains the absolute file
+     * paths of patternFile and MappingFile .
+     * 
+     * @param transformationId
+     *            Transformation ID of the transformation.
+     * @return Returns the transformation file object
+     */
+    @Transactional
+    @Override
+    public ITransformationFile getTransformationFile(String transformationId) {
+
+        TransformFilesDTO transformDTO = (TransformFilesDTO) transformationDAO.getDTO(transformationId);
+
+        String fileLocation = transformationFileService.getTransformFileLocation();
+        String absolutePatternFilePath = getAbsoluteFilePath(fileLocation, transformationId,
+                transformDTO.getPatternFileName());
+        String absoluteMappingFilePath = getAbsoluteFilePath(fileLocation, transformationId,
+                transformDTO.getMappingFileName());
+
+        TransformationFile transformFile = new TransformationFile();
+        transformFile.setAbsolutePatternFilePath(absolutePatternFilePath);
+        transformFile.setAbsoluteMappingFilePath(absoluteMappingFilePath);
+        transformFile.setTitle(transformDTO.getTitle());
+        transformFile.setDescription(transformDTO.getDescription());
+        transformFile.setPatternFileName(transformDTO.getPatternFileName());
+        transformFile.setPatternDescription(transformDTO.getMappingDescription());
+        transformFile.setPatternTitle(transformDTO.getPatternTitle());
+        transformFile.setMappingFileName(transformDTO.getMappingFileName());
+        transformFile.setMappingDescription(transformDTO.getMappingDescription());
+        transformFile.setMappingTitle(transformDTO.getMappingTitle());
+        transformFile.setUserName(transformDTO.getUserName());
+
+        return transformFile;
+    }
+
+    /**
+     * appends the location with dirName and fileName and returns the resultant
+     * string as absoluteFilePath
+     * 
+     * @param location
+     * @param dirName
+     * @param fileName
+     * @return
+     */
+    private String getAbsoluteFilePath(String location, String dirName, String fileName) {
+
+        StringBuffer absoluteFilePath = new StringBuffer();
+
+        absoluteFilePath.append(location);
+        absoluteFilePath.append("/");
+        absoluteFilePath.append(dirName);
+        absoluteFilePath.append("/");
+        absoluteFilePath.append(fileName);
+
+        return absoluteFilePath.toString();
+    }
+
+    @Transactional
+    @Override
+    public List<ITransformation> getTransformations(String transformationIds) {
```

Comment:
This method should get a list of ids, not one string. That you receive one string that need to be split at the commas is controller logic, not business logic.